sw/host/tests/chip/jtag/src/main.rs

```diff
@@ -26,23 +26,15 @@ struct Opts {
     jtag: JtagParams,
 }
 
-fn reset(transport: &TransportWrapper, strappings: &[&str], reset_delay: Duration) -> Result<()> {
-    log::info!("Resetting target...");
-    for strapping in strappings.iter() {
-        transport.pin_strapping(strapping)?.apply()?;
-    }
-    transport.reset_target(reset_delay, true)?;
-    // we want to hold the strapping configuration here because in some life cycle states,
-    // the tap multiplexing is dynamic so remove the tap strap would actually change the tap
-    Ok(())
-}
-
 fn test_openocd(opts: &Opts, transport: &TransportWrapper) -> Result<()> {
     // Reset the device and confirm that its lifecycle state is now RMA by
     // reading the "LCV:" line from the UART.
     let uart = transport.uart("console")?;
 
-    reset(transport, &[], opts.init.bootstrap.options.reset_delay)?;
+    transport.reset_target(
+        opts.init.bootstrap.options.reset_delay,
+        /*clear_uart_rx=*/ true,
```

Comment:
See previous comment, args like this use types to prevent the need for comments.